[#162565305] Change sidebar on ownership listing

software-project opened the pull request:
https://www.pivotaltracker.com/n/projects/1405352/stories/162565305

akegan commented on app/assets/json/translations/locale-en.json:
```diff
@@ -653,7 +656,8 @@
             "SUBMIT_A_PAPER_APPLICATION": "Submit a Paper Application",
-            "VISIT_A_HOUSING_COUNSELOR": "Visit a local housing counselor for help with your application."
+            "VISIT_A_HOUSING_COUNSELOR": "Visit a local housing counselor for help with your application.",
+            "FULFILL_ELIGIBITY_REQUIREMENTS": "Make sure you fulfill the <a href='https://sfmohcd.org/homebuyer-program-eligibility'>eligibility requirements</a> for this listing before you begin."
```
minor nitpick, it looks like elsewhere in locales-en we pass in urls as a variable rather than hardcoding it. 
See e.g. NAV.GET_FEEDBACK

software-project commented:
@akegan I addressed also your refactoring suggestions.

akegan commented:
something funny is going on with the commit history on this one. Looks like the files josh added in the homepage PR are being added again?

akegan commented on app/assets/javascripts/listings/components/process-section.html.slim:
```diff
@@ -69,11 +69,7 @@ accordion-heading
-
-  .content-group ng-if="::$ctrl.parent.listing.Leasing_Agent_Phone || $ctrl.parent.listing.Leasing_Agent_Email"
-    ng-include src="'listings/templates/listing/_contact_leasing_agent.html'"
+  ng-include src="'listings/templates/listing/_additional_info.html'"
```
one nitpick. We have an additional info section in the listing already so this naming might be a bit confusing. 

Maybe it could be additional_sidebar_info?

akegan commented on app/assets/javascripts/listings/templates/listing/_additional_info.html.slim:
```diff
@@ -0,0 +1,24 @@
+.content-group
+  ng-include src="'listings/templates/listing/_what_to_expect.html'"
+
+.content-group ng-repeat="lotteryPreference in ::$ctrl.parent.listing.preferences" ng-if="!$ctrl.listingHasLotteryResults() && lotteryPreference.pdfUrl"
```
I don't know what this actually looks like, but it looks like it's related to the lottery stuff? @callaghanc or @chrisdolendo do you know what this is or how to reproduce this for QA?

akegan replied:
hrm I asked larry about that for the lottery complete bug and I'm not sure that we know how to do that.

software-project commented:
@akegan I renamed additional info partial and rebased off master. Weird files are gone now. Also added review instructions for mobile, but please check if they are correct.

akegan commented on app/assets/javascripts/listings/templates/listing/_additional_sidebar_info.html.slim:
```diff
@@ -0,0 +1,24 @@
+.content-group
+  ng-include src="'listings/templates/listing/_what_to_expect.html'"
+
+.content-group ng-repeat="lotteryPreference in ::$ctrl.parent.listing.preferences" ng-if="!$ctrl.listingHasLotteryResults() && lotteryPreference.pdfUrl"
```
I'm still not sure what this section is. @chrisdolendo do you know what it is? I can't find anything like it in the pattern lib too. I'm tempted to get rid of it.

akegan commented:
@software-project just that one question left, but otherwise code looks good and QA looks ok too.
